symengine/cwrapper.cpp

```diff
@@ -200,6 +205,34 @@ double real_double_get_d(const basic s)
     return (rcp_static_cast<const RealDouble>(s->m))->as_double();
 }
 
+#ifdef HAVE_SYMENGINE_MPFR
+
+void real_mpfr_set_d(basic s, double d, int prec)
+{
+    mpfr_t mp;
```

Comment:
That is to set precision. To set the value, we would anyway have to access the `mpfr_t` directly. Unless we go through the `mpfr_class(std::string s, mpfr_prec_t prec = 53, unsigned base = 10)` class.